static/app/views/settings/project/projectServiceHookDetails.tsx

```diff
@@ -7,203 +7,209 @@ import {
+      <PanelBody withPadding>
+        {!emptyStats ? (
+          <MiniBarChart
+            isGroupedByDate
+            showTimeInTooltip
+            labelYAxisExtents
+            series={[series]}
+            height={150}
+          />
+        ) : (
+          <EmptyMessage
+            title={t('Nothing recorded in the last 30 days.')}
+            description={t('Total webhooks fired for this configuration.')}
+          />
+        )}
+      </PanelBody>
+    </Panel>
+  );
+}
+
+export default function ProjectServiceHookDetails() {
+  const organization = useOrganization();
+  const {hookId, projectId} = useParams<{hookId: string; projectId: string}>();
+  const api = useApi();
+  const navigate = useNavigate();
+
+  const {
+    data: hook,
+    isPending,
+    isError,
+    refetch,
+  } = useApiQuery<ServiceHook>(
+    [`/projects/${organization.slug}/${projectId}/hooks/${hookId}/`],
+    {staleTime: 0}
+  );
+
+  const deleteMutation = useMutation({
+    mutationFn: () => {
+      return api.requestPromise(
+        `/projects/${organization.slug}/${projectId}/hooks/${hookId}/`,
+        {
+          method: 'DELETE',
+        }
+      );
+    },
+    onMutate: () => {
+      addLoadingMessage(t('Saving changes\u2026'));
+    },
+    onSuccess: () => {
+      clearIndicators();
+      navigate(
+        normalizeUrl(`/settings/${organization.slug}/projects/${projectId}/hooks/`)
+      );
+    },
+    onError: () => {
+      addErrorMessage(t('Unable to remove application. Please try again.'));
+    },
+  });
+
+  if (isPending) {
+    return <LoadingIndicator />;
+  }
+
+  if (isError) {
+    return <LoadingError onRetry={refetch} />;
+  }
+
+  if (!hook) {
+    return null;
   }
+
+  return (
+    <Fragment>
+      <SettingsPageHeader title={t('Service Hook Details')} />
+
+      <ErrorBoundary>
+        <HookStats />
+      </ErrorBoundary>
+
+      <ServiceHookSettingsForm
+        organization={organization}
+        projectId={projectId}
+        hookId={hookId}
+        initialData={{
+          ...hook,
+          isActive: hook.status !== 'disabled',
+        }}
+      />
+      <Panel>
+        <PanelHeader>{t('Event Validation')}</PanelHeader>
+        <PanelBody>
+          <PanelAlert type="info" showIcon>
+            Sentry will send the <code>X-ServiceHook-Signature</code> header built using{' '}
+            <code>HMAC(SHA256, [secret], [payload])</code>. You should always verify this
+            signature before trusting the information provided in the webhook.
+          </PanelAlert>
+          <FieldGroup
+            label={t('Secret')}
+            flexibleControlStateSize
+            inline={false}
+            help={t('The shared secret used for generating event HMAC signatures.')}
+          >
+            <TextCopyInput>
+              {getDynamicText({
+                value: hook.secret,
+                fixed: 'a dynamic secret value',
+              })}
```

Comment:
can rip these out if you see them
```suggestion
              {hook.secret}
```